doc/src/set.rst

```diff
@@ -183,9 +187,13 @@ properties to reset and what the new values are.  Some strings like
 This section describes how to select which atoms to change
 the properties of, via the *style* and *ID* arguments.
 
+.. versionchanged:: 28Mar2023
+
+   Support for type labels was added for selecting atoms by type
+
 The style *atom* selects all the atoms in a range of atom IDs.  The
-style *type* selects all the atoms in a range of types.  The style
-*mol* selects all the atoms in a range of molecule IDs.
+style *type* selects all the atoms in a range of types or type labels.
```

Comment:
Adding "or type labels" here is somewhat redundant, as stated. Atoms are still just selected by type, this PR just allows type labels to do that (as is clarified above).